specification/search/data-plane/Microsoft.Azure.Search.Service/stable/2019-05-06/searchservice.json

```diff
@@ -3906,6 +3911,28 @@
       "x-nullable": false,
       "description": "Represents the overall indexer status."
     },
+    "IndexerLimits": {
+      "properties": {
+        "maxRunTime": {
+          "type": "string",
+          "format": "duration",
+          "readOnly": true,
+          "description": "The maximum duration that the indexer is permitted to run for one execution."
+        },
+        "maxDocumentExtractionSize": {
+          "type": "number",
+          "format": "int64",
+          "readOnly": true,
+          "description": "The maximum size of a document which will be considered valid for indexing."
```

Comment:
Please specify units in the description. To make it more self documenting, consider using x-ms-client-name so you can include the units in the name in the generated code.

Reply:
I'll just add the units in the description to be consistent with everywhere else in the spec!